### ISSUE TITLE
Visualisation of the data flows

### ISSUE THREAD

**Issue by kvoer** [stichtingsem/ecosystem#10]
### Can we create a visual representation of the data flows of the critical data objects `entitlement` and `learning material list` to clarify the working of the SEM Ecosystem Standard?

Suggestion is to use a state diagram or a sequence diagram?

**Comment by TM-Frank** [comment]
Other userfull diagrams:

1. entity-relation diagram
All datacomponents and fields, names en relationships in one view, will give some good insights.

2. Ownership of data-entities and ownerships transistions.
This has to do which party has to grant which data and when.

**Comment by kvoer** [comment]
Within the functional track we addressed this issue and created:
1. New overall proces flow of all exchange activities within the ecosystem structured by role and main process
2. For every main process we developed a detailed proces flow including the required and exchanged data objects within the ecosystem
3. A clear and unambiguous definition of an entitlement
4. A specification of four variants for the entitlement
5. The different states in the lifecycle of an entitlement

These deliverables are currently under review.
